src/components/cart/cart.tsx

```diff
@@ -23,6 +23,8 @@ export const Cart = ({ className, initialIsOpen }: CartProps) => {
         if (initialIsOpen !== undefined) {
             setIsOpen(initialIsOpen);
         }
+        // ignore `initialIsOpen` changes, cause it is only for initial state
```

Comment:
useCartOpen can't utilize it, cause it just using a context...
in general, initialIsOpen is only used in boards, so we just might add this to a list instead of ignoring.
also, we have to consider refactoring this components I believe

Reply:
if it's used only in boards let's then remove the prop from component and set initial value to true in the context just for boards

Reply:
moved this to context